modules/monitoring-about-cluster-monitoring.adoc

```diff
@@ -0,0 +1,17 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/understanding-the-monitoring-stack.adoc
+
+[id="about-monitoring_{context}"]
+= About monitoring
+
+[role="_abstract"]
+{product-title} includes a pre-configured, pre-installed, and self-updating monitoring stack that provides *monitoring for core platform components*. {product-title} delivers monitoring best practices out of the box. A set of alerts are included by default that immediately notify cluster administrators about issues with a cluster. Default dashboards in the OpenShift Container Platform web console include visual representations of cluster metrics to help you to quickly understand the state of your cluster.
+
+After installing {product-title}, cluster administrators can optionally enable *monitoring for user-defined projects*. By using this feature, cluster administrators, developers, and other users can specify how services and pods are monitored in their own projects. You can then query metrics, review dashboards, and manage alerting rules and silences for your own projects in the OpenShift Container Platform web console.
```

Comment:
Is this for OSD/ROSA specifically? I would suggest removing this if so.

Reply:
@fahlmant  I removed the text. Should I also remove the note: "Cluster administrators can grant developers and other users permission to monitor their own projects. Privileges are granted by assigning one of the predefined monitoring roles."?

Reply:
Users in the dedicated-admins group specifically can do this

Reply:
OK, the entire topic is now removed. Thanks.